setup.py

```diff
@@ -219,7 +219,7 @@
 
 setup_kwargs = {
     "name": "pymedphys",
-    "version": "0.34.0.dev2",
+    "version": "0.34.0.dev3",
```

Comment:
inconsistent with semver and above representations, which are 0.34.0-dev3

Reply:
> setup.py is inconsistent with semver.
> If that is necessary, play through.
> Urge review of the other comments though.

Thanks for the review :). I really, really appreciate these. Thank you. Making my dream of peer review in medical physics open source software a reality :). Thank you.

Reply:
For this one, this is actually the Python default... And it is what poetry autogenerates. So I'll leave this one here.